src/components/Dropdown/Dropdown.scss

```diff
@@ -0,0 +1,25 @@
+@use "../../assets/theme/macbis-colors" as c;
+
+.usa-combo-box {
+  .usa-combo-box__input:read-only {
+    cursor: pointer;
+  }
+  .usa-combo-box__input-button-separator {
+    line-height: calc(2.5rem - 18px);
```

Comment:
Never actually seen the calc css function in action before! Pretty sweet

Reply:
Oh yeah, it's a real nice one to have especially when you're trying to be precise with relative units.